moscar.go

```diff
@@ -1,124 +0,0 @@
-package main
-
-import (
-	"fmt"
-	"github.com/micromdm/squirrel/munki/datastore"
-	"github.com/micromdm/squirrel/munki/models"
-	"html/template"
-	"io"
-	"log"
-	"net/http"
-	"time"
-)
-
-const STATIC_URL string = "/static/"
-const STATIC_ROOT string = "static/"
-const REPO_PATH string = "/Users/Shared/repo"
-
-type Context struct {
-	Static string
-	Prods  []ParsedPkginfo
-}
-
-// Template expects icon, 'product' name, description, and version, which becomes the download URL
-type ParsedPkginfo struct {
-	Icon        string
-	Name        string
-	Descript    string
-	Version     string
-	DownloadURL string
-}
-
-func Parse(in_pkginfo *models.PkgsInfo, ppi *ParsedPkginfo) *ParsedPkginfo {
```

Comment:
Inside `Home()` you have 
```
	for _, pkginfo := range allcat {
		pi := &ParsedPkginfo{}
		ppi := Parse(pkginfo, pi)
```
You're initializing an empty struct outside and then passing it in to be modified. Sometimes that's a valid thing to do, but in this case it's not necessary. A cleaner way to rewrite this function would be:

```
func parse(input *munki.PkgsInfo) *ParsedPkginfo {
	return &ParsedPkginfo{
		Icon:        input.IconName,
		Name:        input.Name,
		Descript:    input.Description,
		Version:     input.Version,
		DownloadURL: input.DownloadURL,
	}
}
```